Tests/UnitTests/Core/Vertexing/VertexingDataHelper.hpp

```diff
@@ -44,12 +44,15 @@ struct VertexInfo {
 };
 
 std::tuple<
-    Vertex<BoundParameters>, std::vector<VertexInfo>,
+    Vertex<BoundTrackParameters>, std::vector<VertexInfo>,
     std::vector<
-        BoundParameters>> inline readTracksAndVertexCSV(std::string toolString,
-                                                        std::string fileBase =
-                                                            "vertexing_event_"
-                                                            "mu20") {
+        BoundTrackParameters>> inline readTracksAndVertexCSV(std::string
+                                                                 toolString,
+                                                             std::string
+                                                                 fileBase =
+                                                                     "vertexing"
+                                                                     "_event_"
+                                                                     "mu20") {
```

Comment:
Can you maybe switch clang format off for this part? This is pretty terrible :)

Reply:
Fixed it. This actually happened because the `inline` keyword came after the return type instead of before. I thought that would not compile.